pkg/flink/handler.go

```diff
@@ -236,6 +238,18 @@ func flinkClusterTaskInfo(ctx context.Context, flinkCluster *flinkOp.FlinkCluste
 	}, nil
 }
 
+func isSubmitterExitCodeRetryable(ctx context.Context, exitCode int32) bool {
+	logger.Infof(ctx, "job submitter failed: %v", exitCode)
```

Comment:
🎨 
```suggestion
	logger.Infof(ctx, "job submitter failed: %d", exitCode)
```

Reply:
otoh, I think I would not log under this helper func and defer the logging to where it will be more meaningful. i.e depending on the usage context the above msg might not make sense.